Cargo.toml

```diff
@@ -83,7 +83,7 @@ untrusted = "0.6"
 
 [target.'cfg(target_os = "linux")'.dependencies]
 libc = "0.2"
-inotify = { git = "https://github.com/inotify-rs/inotify" }
+inotify = { git = "https://github.com/hawkw/inotify", branch = "eliza/fix-bad-api" }
```

Comment:
Would you mind renaming this branch to something less judgey? I'd prefer to avoid even the appearance of picking on the inotify project, especially since they've very amenable to feedback in the past.

Reply:
Or, should we just depend on a revision so that the build doesn't depend on the branch's existence?

Reply:
we should probably depend on a revision, but I'll rename the branch regardless. fwiw, the `inotify` maintainer also seems to think this API is bad, but yeah, I can see how it comes off as too judge-y.